frontend_tests/node_tests/dispatch.js

```diff
@@ -498,10 +498,17 @@ run_test("realm_bot remove", ({override}) => {
     admin_stub.get_args("update_user_id", "update_bot_data");
 });
 
-run_test("realm_bot delete", () => {
+run_test("realm_bot delete", ({override}) => {
     const event = event_fixtures.realm_bot__delete;
+    const bot_stub = make_stub();
     // We don't handle live updates for delete events, this is a noop.
```

Comment:
Do we need to change or remove the comment above?

Reply:
Oops, I forgot to remove this, removed now.